### PR TITLE
[React] Row Actions Topic

### PR DESCRIPTION
**Pull request by tishko0**

### CODE REVIEW

File: doc/en/components/grids/_shared/row-actions.md [hunk 1]
```diff
@@ -76,6 +76,19 @@ They are added inside the `{ComponentName}` and this is all needed to have an `A
 ```
 <!-- end: WebComponents -->
 
+<!-- React -->
+```html
```
Comment (ChronosSF) [inline doc/en/components/grids/_shared/row-actions.md:80]: needs to be changed to ```tsx

File: doc/en/components/grids/_shared/row-actions.md [hunk 2]
```diff
@@ -133,6 +146,17 @@ These components expose templates giving flexibility for customization. For inst
...
```
Comment (ChronosSF) [inline doc/en/components/grids/_shared/row-actions.md:150]: needs to be changed to ```tsx

**Review by ChronosSF — CHANGES_REQUESTED** [review]

**Review by ChronosSF — CHANGES_REQUESTED** [review]
You should double check this document. The second header ```## Custom Implementation``` is only visible for Angular because of a condition. This means that the Blazor code snippet is lost and the WC and React codesnippets don't have any section and are just appended to Usage.

In addition the WC and React code snippets kind of miss the point. They simply show the editing actions props while the idea of the paragraph is to show custom buttons like in the Blazor and Angular snippets.

**Comment by IMinchev64** [comment]
> You should double check this document. The second header `## Custom Implementation` is only visible for Angular because of a condition. This means that the Blazor code snippet is lost and the WC and React codesnippets don't have any section and are just appended to Usage.
> 
> In addition the WC and React code snippets kind of miss the point. They simply show the editing actions props while the idea of the paragraph is to show custom buttons like in the Blazor and Angular snippets.

Currently, only Angular supports the custom implementation, which is why only Angular has such a section in the documentation topics for now. The reason behind this limitation is that the required CRUD service is not yet exported, which prevents creating a custom button with an associated click event for the other platforms.

File: doc/en/components/grids/_shared/row-actions.md [hunk 3]
```diff
@@ -76,6 +76,19 @@ They are added inside the `{ComponentName}` and this is all needed to have an `A
 ```
 <!-- end: WebComponents -->
 
+<!-- React -->
+```tsx
+<{ComponentName} id="grid" rowEditable="true" primaryKey="ID">
```
Comment (mddragnev) [inline doc/en/components/grids/_shared/row-actions.md:81]: This should be ComponentSelector like the next react snippet below

**Review by mddragnev — DISMISSED** [review]

**Review by ChronosSF — APPROVED** [review]

**Pull request merged**